src/async_impl/client.rs

```diff
@@ -1239,6 +1247,22 @@ impl ClientBuilder {
         self.config.dns_overrides.insert(domain.to_string(), addr);
         self
     }
+
+    /// Override DNS resolution for specific domains to particular IP addresses.
+    ///
```

Comment:
Maybe document that the function will not be called every time, because results will be cached (if that is correct).

Reply:
@niuhuan With "if that is correct" I wanted to say that I'm not entirely sure how reqwest works, maybe I am wrong. But you can remove that quote from your commit.

Reply:
🤔   I tried multiple requests, only one domain name resolution, even if the client is cloned. client must rebuild.

Reply:
I am not proficient in English, please reply to me if the description is incorrect.